package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "small-client-wiki",
+  "version": "1.0.0",
+  "description": "Read-Only React Federated Wiki Browser",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/WardCunningham/small-client-wiki.git"
```

Comment:
npm interprets that strange arithmetic.
https://docs.npmjs.com/files/package.json

![screen shot 2016-11-14 at 8 07 38 pm](https://cloud.githubusercontent.com/assets/12127/20292873/12d4b75a-aaa6-11e6-99d7-5715510d3d07.png)